src/connectors/netease-music.js

```diff
@@ -0,0 +1,54 @@
+import BaseConnector from 'content/base-connector';
+import Utils from 'content/utils';
+import _ from 'underscore';
+
+new class extends BaseConnector {
+    constructor() {
+        super();
+
+        this.name = 'Netease.Music';
+        this.prefix = '/com/163/music';
+
+        this.artistSelector = '.by.f-thide.f-fl > span';
+        this.albumSelector = '.j-flag.words > a[class*="src"]';
+        this.artSelector = '.head.j-flag > img';
```

Comment:
Album art URL has a size parameter (something like `?param=34y34`) at the end. Removing it results in a bigger image. You could see similar behaviour in SoundCloud connector.